audit/projects/k8s-infra-e2e-boskos-001/services/compute/project-info.json

```diff
@@ -3,7 +3,7 @@
est2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC1HW5T8W/GHe9ba3zZhCR59IjYskzVSoR9KQQF97qG37mPsgvp+HZaFN4czh5Nq5SL7EbCZsROj50cMaO4Q37wCKvIQoWQtaFnBZkzvcADqPAyvIkGjsVRP7+3bSW+2vFr1bqmE8X5lFvIrAdR6BFbgirfTdeOT4SZo6OzyXm4mznYFQMxz5MIqNA+64qMq7UKWKYULRI33YMn8kvUVi3/sATPjOt4v0lQAM8i0g6IKw9MYlLNBQ9G7nH3tYWBBXAMboOT5TsGKGSt8FooNhxKuygbSyJoQL9T5x24sDjxqOVMvtTLRk3NtigdIcigiFgbRxGSmoNXGg9OIIP5Bcyh root@df1a6954-82ae-11eb-b915-56a8d6cedae4\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDfySXJDe/9NLf9ek6OEu5CKV4ugzXnI4I+Ealg9fqaB2+S0BCsSGnrCOGUjKH0PWy85VqwhIhd+rTPz3sZRsbWTVvtnTWgLzBFnC20TuJlhmj+EUIIJcigSrkQcOOpGJi2trDBr1eXAN6SY9ZqHQRicndDVsJ9oi9eXrTUYoSwM+gbIWxscrRdvnsT61m1wLSmxJOoEb9013ow66j+RhDRd1MtJrrF1qoKOIPo9FtVHoOJuZ8gV2mASemGGx1xiTZSboDubvkKVroGzhG+uVEXaTqh7GJsmdN8yvjw/9qfphU7ihDuqZCIpiFx3HNV4tYsA6h1DhD92E7mInGC7Zsn root@d594c422-84c1-11eb-b098-eeb808a18dc8\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDhsM5onslgH0nRFnAumVOGK9oHjF1t3VduBnRW5e3XgBu/qxoP3eT7qHYozRGTFLOrHPWIJ1ScyOEfV8cTT9aVNVamk3/5+UR34yOxQR/L5mrDyOd9o8hBvQ3BJiM6S775zQ4sy+oKTL9MMoT3m8/tC8n+lS+SNHyVFQXQmFEOLdiMN+qKenkYt5cv+lIzqKXUxtiJ5GQOsK2mY2TC+JtyeDSeCEjv1e/bmiEkWmiG1e550VdPffBfD0DOp5DWgkGM0a72SW1mPxjaxUy4ttk7JEPA76WiuLwG/mc6t28R5fmklQOyYK1LppGX1NhhLSGtwJ/abN8rgyWJqDyny+43 root@ad4d54b9-8af8-11eb-8155-227e2855667f\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDIciYtiBpbHldYdy3OXDuSRbR4SgnK/kzx3fVYKb6+HCyiPBFEPON6xuxBTgb2i5vwuWH4tq7zA25+8we95Y81DSUaPs3kOMS+m03Nwg76LmYZxZTF7HTCp72QMSFZFPp0hu+IZertWcJ21CrnA9nxOd/XfCS+ZgtbCDZW4MrUP4Dx/Gs1dUyqqla/0GvU7StQjwGHGs7j+59w5W2oYrK75pW1VETikPNISAKorNE9U6yOGmQHk9Iw4qkY3UpGYSB2wKLuNAH6dVPBxEkW+PqL8jiyaXsKWA8WGnxSZtm2LUuXbEJ7Va16AVkzz0NLecBaCtR3q+CL1fOoDO0wlqlr root@3c8b1b38-8c41-11eb-b692-3a35d43432cb\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDFpnDgq2z7HFkMOm5cFxWC/ygaQZz6WIr/ADtBlkGNdfhXfQXOwPx6I9CuaUApj+6EmSUMWOHFJ+2vBqL7n06OEmua1zhNjYJxjyI9Fovektew6idwcXYntUmWoYusoVyitI8JLIh9qwuOp7JHDzIw3e28o6k+z8VliVNoe2k+O516x3CTlaYFpnJfxXi1YmCAPuq0i1YYNqJ6AvA1axHUeq0YYw3ruNriNsENlgwVUO3HP+JjxI5PIYZHFv/ZYvYgcXBfDMrT29BiC1CTM5dVtT9OstMxtSSOR2JB5PTN/FukNigdne2APgG00p/QZcNIDvVCoDgNI+mTrDdAMVfd root@3599990f-8e54-11eb-adda-5eb174036857\nkubetest2:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDGwOPOG1FFup69yB/68lDCeIz1/outPDVIpyhaPZhxpy11w3wp/XFbaS807TNUKRs0vD43vxX3U2OS+hl4g+8p+uRdT9bVoFm5+PqxsDuSVLI3Ch5/itCo+Ux37wLLda9eBPqHC7fvgNyFqly00MeviZr1mooxgdxDOgLb4UHtKI9ke9QXlcxBkkEcWFZe6KLM5spcau5N9mhMWJryoUfwBHY6L/zcdIC0+YtUCT4Gz94J3YHv/ADDYfo73r5IfDJHXFqEXfZo++4nGIjFsW32emLJW9+S99GkDrnQEjUpwiybhhDklsiFe98/+oX7iDJcvzsirhGtZnMnj6FoCJEV root@a9e3680e-914c-11eb-827a-4615317e4d96\nkubetest2:prow:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCmYxHh/wwcV0P1aChuFLpl28w6DFyc7G5Xrw1F8wH1Re9AdxyemM2bTZ/PhsP3u9VDnNbyOw3UN00VFdumkFLjLf1WQ7Q6rZDlPjlw7urBIvAMqUecY6ae1znqsZ0dMBxOuPXHznlnjLjM5b7O7q5WsQMCA9Szbmz6DsuSyCuX0It2osBTN+8P/Fa6BNh3W8AF60M7L8/aUzLfbXVS2LIQKAHHD8CWqvXhLPuTJ03iSwFvgtAK1/J2XJwUP+OzAFrxj6A9LW5ZZgk3R3kRKr0xT/L7hga41rB1qy8Uz+Xr/PTVMNGW+nmU4bPgFchCK0JBK7B12ZcdVVFUEdpaAiKZ prow"
```

Comment:
The first and last entries in here look identical.  Which makes me think kubetest2 is starting to reuse keys.  But I'm not clear why it didn't pick up the first entry.